people/service.go

```diff
@@ -112,22 +113,34 @@ func (s *peopleServiceImpl) getPersonByUUID(uuid string) (person, bool, error) {
 
 }
 
+func (s *peopleServiceImpl) openDB() error {
+	if s.db == nil {
+		var err error
+		if s.db, err = bolt.Open(s.cacheFileName, 0600, &bolt.Options{Timeout: 1 * time.Second}); err != nil {
+			log.Errorf("ERROR opening cache file for init: %v", err.Error())
+			return err
+		}
+		return s.createCacheBucket()
```

Comment:
you could get rid of it, and fall over to the outside return (but I don't have a strong view on that)